src/compiler/parser.ts

```diff
@@ -345,7 +345,7 @@ module ts {
     module Parser {
         // Share a single scanner across all calls to parse a source file.  This helps speed things
         // up by avoiding the cost of creating/compiling scanners over and over again.
-        const scanner = createScanner(ScriptTarget.Latest, /*skipTrivia:*/ true);

```

Comment:
The comment is supposed to resemble a 'named parameter'.  That's how they were originally introduced.  And it means they're easy to search for as well.  I would keep the colon.